test/raw_data_gis/scripts/test_raster_bio1clip.json

```diff
@@ -0,0 +1,49 @@
+{
+    "archived": "zip",
+    "citation": "",
+    "dataset_count": "1",
+    "datum": "",
+    "description": "bio1clip.tif",
+    "driver": "GTiff",
+    "file_size": "",
+    "group_count": "",
+    "homepage": "fill",
+    "keywords": [],
+    "licenses": [],
+    "name": "test-raster-bio1clip",
+    "projection": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\",EAST],AUTHORITY[\"EPSG\",\"4326\"]]",
+    "resources": [
+        {
+            "band_name": "bio1clip_1",
+            "color_table": null,
+            "extensions": [
+                "tif"
+            ],
+            "format": "raster",
+            "name": "bio1clip",
+            "no_data_value": -9999.0,
+            "other_paths": "",
+            "path": "bio1clip.tif",
+            "scale": null,
+            "statistics": {
+                "maximum": 0.0,
+                "mean": 0.0,
+                "minimum": 0.0,
+                "stddev": -1.0
+            },
+            "url": "https://github.com/henrykironde/retriever/blob/testgdal2/test/raw_data_gis/test_raster_bio1clip.zip"
```

Comment:
I think this url is at fault here, if you try to make a get request at this url, it returns the html code of the webpage. The correct url would be : https://github.com/henrykironde/retriever/raw/testgdal2/test/raw_data_gis/test_raster_bio1clip.zip